### DISCUSSION TITLE
Unable to certify student qualifications

### DISCUSSION THREAD

**Discussion by dentar142** [community/community#141188] [GitHub Education]
I am a student at Fuzhou University. I had a problem verifying my student qualification. He repeatedly reminded me that the documents I uploaded did not meet the requirements, that my personal information did not match and that I was not on campus. Well, I hope the official staff can solve this problem for me,thank you in advance.

**Comment by CemWebDev** [comment]
Hello, I understand the frustration with the verification process. One thing that might help is using the GitHub mobile app and linking it to your account. After connecting your account through the app, try submitting your verification documents again. This method has worked for me.

I hope this helps. Good luck!

**Reply by dentar142** [reply]
Thanks for your advice,that's very nice of you to told me so.

**Reply by RaphaelAsla** [reply]
Did you need to change your account's username ?
I am not sure if I am understanding this correctly but it looks like my application is getting denied because of that reason, not having my full name as a username.
At least that's what I understand from the feedback email.
But of course I can't change change it, every link to my repos will stop working...

**Reply by dentar142** [reply]
Err,for the officially speaking that you should change your name on public profile and payment name should as same as the name on your unniversity document.For me it's my document has a little problem and my location couldn't be detected in the right place.

**Comment by CristyNel** [comment]
Ensure you have  set 2 authentication and your correct Name, they will not accept your verification before that.
